src/bench/bech32.cpp

```diff
@@ -10,12 +10,13 @@
 #include <string>
 #include <vector>
 
+using util::HexLiteral;
 
 static void Bech32Encode(benchmark::Bench& bench)
 {
-    std::vector<uint8_t> v = ParseHex("c97f5a67ec381b760aeaf67573bc164845ff39a3bb26a1cee401ac67243b48db");
+    constexpr std::array<uint8_t, 32> v{HexLiteral<uint8_t>("c97f5a67ec381b760aeaf67573bc164845ff39a3bb26a1cee401ac67243b48db")};
```

Comment:
In commit "refactor: Hand-replace some ParseHex -> util::HexLiteral" (288f995f71fd8e6f750d668e47237e64d893cc1f)

I feel like commit message could benefit from a summary of the changes it is making. Maybe would suggest:

- The following scripted-diff commit will replace `ParseHex(...)` with `HexLiteral<uint8_t>(...)` but replacement will not work in cases where vectors are needed instead of arrays, and is not ideal in cases where std::byte can be used instead of uint8_t, so make replacements in these cases manually.